lib/subject_selector.rb

```diff
@@ -3,47 +3,38 @@ class MissingParameter < StandardError; end
 
   attr_reader :user, :params, :workflow
 
-  def initialize(user, workflow, params, scope, session)
-    @user, @workflow, @params, @scope, @session = user, workflow, params, scope, session
+  def initialize(user, workflow, params, scope)
+    @user, @workflow, @params, @scope = user, workflow, params, scope
   end
 
   def queued_subjects
     raise workflow_id_error unless workflow
-    user_enqueued = UserSubjectQueue
-                    .find_by!(user: user.user, workflow_id: params[:workflow_id])
-    selected_subjects(user_enqueued.next_subjects(default_page_size))
-  end
+    raise group_id_error if needs_set_id? 
```

Comment:
Current cellect behaviour is to [select a group randomly](https://github.com/zooniverse/Cellect/blob/0793d08f82d7ad88c7b4d24a0f4eabc1bb2e2b3d/lib/cellect/server/grouped_workflow.rb#L26), do you think we should add this behaviour?